internal/runners/scripts/scripts.go

```diff
@@ -32,39 +30,19 @@ type scriptLine struct {
 	Description string `json:"description,omitempty"`
 }
 
-type outputFormat []scriptLine
-
-func (f outputFormat) MarshalOutput(format output.Format) interface{} {
-	if format == output.PlainFormatName {
-		return f.formatScriptsList()
-	}
-	return f
+type scriptsTable struct {
+	rows []scriptLine
 }
 
-// formatScriptsList formats a lists of all of the scripts defined for this project.
-func (f outputFormat) formatScriptsList() string {
-	hdrs, rows := f.scriptsTable()
-	t := gotabulate.Create(rows)
-	t.SetHeaders(hdrs)
-	t.SetAlign("left")
-
-	return t.Render("simple")
-}
-
-func (f outputFormat) scriptsTable() (hdrs []string, rows [][]string) {
-	for _, s := range f {
-		row := []string{
-			s.Name, s.Description,
+func (t scriptsTable) MarshalOutput(format output.Format) interface{} {
```

Comment:
You don't need `MarshalOutput` here. This is plain controller logic, no custom marshaller required.